gui/default/syncthing/core/logViewerModalView.html

```diff
@@ -0,0 +1,36 @@
+<modal id="logViewer" icon="book" status="default" heading="{{'Logs' | translate}}" large="yes" closeable="yes">
+  <div class="modal-body">
+    <ul class="nav nav-tabs">
+      <li class="active"><a data-toggle="tab" href="#log-viewer-log" translate>Log</a></li>
+      <li><a data-toggle="tab" href="#log-viewer-facilities" translate>Debugging Facilities</a></li>
+    </ul>
+    <div class="tab-content">
+
+      <div id="log-viewer-log" class="tab-pane in active">
+        <label translate ng-if="logging.logEntries.length == 0">Loading...</label>
+        <textarea ng-focus="logging.paused = true" ng-blur="logging.paused = false" id="logViewerText" class="form-control" rows="20" ng-if="logging.logEntries.length != 0" readonly style="font-family: Consolas; font-size: 11px; overflow: auto;">{{ logging.content() }}</textarea>
+        <p translate class="help-block" ng-style="{'visibility': logging.paused ? 'visible' : 'hidden'}">Log tailing paused. Click here to continue</p>
```

Comment:
The click on me as text seems fine and better than the new "Click anywhere..." proposal: Otherwise someone might click on the background and complain when the log modal disappears :P 
I would add a dot (or three) at the end though.

Reply:
Three dots to existing text?

Reply:
To be consistent (first sentence gets a final dot): "Log tailing paused. Click here to continue."  
On second thought, I don't like three dots ("continue..."). The idea was, that it signifies that something (trailing) will continue when you click, but that seems unnecessary and might be a problem for translation.